Modules/_sqlite/cursor.c

```diff
@@ -890,6 +890,11 @@ PyObject* pysqlite_noop(pysqlite_Connection* self, PyObject* args)
 
 PyObject* pysqlite_cursor_close(pysqlite_Cursor* self, PyObject* args)
 {
+    if (self->connection == NULL) {
```

Comment:
Style nit: I think `!foo` is more common than `foo == NULL` in sqlite3 codebase. It would be nice to keep styling consistent within the module.